besu-plugins/linea-sequencer/sequencer/src/main/java/net/consensys/linea/sequencer/txselection/selectors/TransactionEventSelector.java

```diff
@@ -0,0 +1,48 @@
+package net.consensys.linea.sequencer.txselection.selectors;
+
+import java.util.Set;
+import java.util.concurrent.atomic.AtomicReference;
+import lombok.RequiredArgsConstructor;
+import net.consensys.linea.bundles.TransactionBundle;
+import org.hyperledger.besu.evm.log.Log;
+import org.hyperledger.besu.plugin.data.TransactionProcessingResult;
+import org.hyperledger.besu.plugin.data.TransactionSelectionResult;
+import org.hyperledger.besu.plugin.services.txselection.PluginTransactionSelector;
+import org.hyperledger.besu.plugin.services.txselection.TransactionEvaluationContext;
+
+@RequiredArgsConstructor
+public class TransactionEventSelector implements PluginTransactionSelector {
+  private final AtomicReference<Set<TransactionEventFilter>> deniedEvents;
+  private final AtomicReference<Set<TransactionEventFilter>> deniedBundleEvents;
+
+  @Override
+  public TransactionSelectionResult evaluateTransactionPreProcessing(
+      final TransactionEvaluationContext evaluationContext) {
+    return TransactionSelectionResult.SELECTED;
+  }
+
+  @Override
+  public TransactionSelectionResult evaluateTransactionPostProcessing(
+      final TransactionEvaluationContext evaluationContext,
+      final TransactionProcessingResult processingResult) {
+    final boolean isBundle =
```

Comment:
not blocking: I think it make sense to add a `isPartOfBundle` to `PendingBundleTx`